xml/System.Runtime.Loader/AssemblyLoadContext.xml

```diff
@@ -239,8 +240,8 @@ This constructor is protected. It can only be called from the <xref:System.Runti
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Represents whether the AssemblyLoadContext is collectible.</summary>
+        <value>If not specified in the constructor, the value is <see langword="false" />.</value>
```

Comment:
```suggestion
        <value><see langword="true" /> if this <see cref="T:System.Runtime.Loader.AssemblyLoadContext" /> is collectible; otherwise, <see langword="false" />. If a value is not specified in the constructor, the value is <see langword="false" />.</value>
```